teps/0048-task-results-without-results.md

```diff
@@ -491,7 +523,7 @@ spec:
     - name: build-image
       runAfter: [ "check-pr-content" ]
       when:
-        - input: "$(tasks.check-pr-content.results.image-change)"
+        - input: "$(tasks.check-pr-content.results.image-change || no)"
```

Comment:
I understand this, but it doesn't look very useful.
I think default results will be useful to execute tasks which depend on tasks which are controlled by a when expression, rather than the when expression directly. There is no reason why the "Check PR content" result should be missing. Instead, if the build-image is not executed, following tasks may won't a default value for the image to be used.